### ISSUE TITLE
KeyboardAvoiderInsets extraSpace prop should default to 0 instead of 20

### ISSUE THREAD

**Issue by soemarko** [iway1/react-native-keyboard-avoider#11]
https://github.com/iway1/react-native-keyboard-avoider/blob/3d446aaf150bb8cbcd7366b9620acd52fec69de0/package/src/components/common-props.ts#LL22C5-L22C19 this line.

The semantics "extra space" should be additional spacing. Took me awhile to debug why there are a huge gap between the text input and the keyboard.

**Comment by merveillevaneck** [comment]
I agree with the above. I'd also like to add that for me, changing the extraSpace prop value at all doesnt even work in expo. I'm not sure why and its quite hard to isolate. But im completely unable to remove the gap between the keyboard and the view. in spite of there being a prop for it.
